example/gamemode.ini

```diff
@@ -47,6 +47,16 @@ inhibit_screensaver=1
 ; The DRM device number on the system (usually 0), ie. the number in /sys/class/drm/card0/
 ;gpu_device=0
 
+; VSync settings
+; Only works if application is started with gamemoderun
+; Allowed options: none, force_disable, force_enable
+;vsync_mode=none
+
+; Hybrid GPU settings
+; Only works if application is started with gamemoderun
+; Allowed options: prime, bumblebee, primus, nvidia-xrun
+;hybrid_gpu_mode=none
+
```

Comment:
Since these aren't related to `gpu_device` and the nvidia settings, it'd be better to keep them separate and away from the here be dragons warning, as these are pretty safe.

Reply:
That's a good idea, I'm going to change that.